bw2io/export/csv.py

```diff
@@ -153,10 +153,46 @@ def get_activity(self, act):
         return data
 
     def get_unformatted_data(self):
-        """Return all database data as a nested dictionary:
-
-        .. code-block:: python
+        # """Return all database data as a nested dictionary:
+
+        # .. code-block:: python
+
+        #     {
+        #         'database': {
+        #             'name': name,
+        #             'metadata': [(key, value)],
+        #             'parameters': {
+        #                 'columns': [column names],
+        #                 'data': [[column values for each row]]
+        #             },
+        #             'project parameters': {
+        #                 'columns': [column names],
+        #                 'data': [[column values for each row]]
+        #             }
+        #         },
+        #         'activities': [{
+        #             'name': name,
+        #             'metadata': [(key, value)],
+        #             'parameters': {
+        #                 'columns': [column names],
+        #                 'group': 'group name',
+        #                 'data': [[column values for each row]]
+        #             },
+        #             'exchanges': {
+        #                 'columns': [column names],
+        #                 'data': [[column values for each row]]
+        #             }
+        #         }]
+        #     }
+
+        # """
+        """
+        Return all database data as a nested dictionary:
 
+        Returns
+        -------
+        dict
+            A nested python dictionary with the following structure:
             {
```

Comment:
Does the code render correctly locally? I guess it should, [if it is indented](https://numpydoc.readthedocs.io/en/latest/format.html#other-points-to-keep-in-mind).